From 59c097b45d836ff665ee3b944f0e60a3e2a42403 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Thu, 09 Aug 2018 06:21:36 +0300
Subject: [PATCH] t32 [scripts]: update program flash script

Do SW reset to make sure we start with clean SPI bus
---

diff --git a/board/habanalabs/common/t32/prog_spi.cmm b/board/habanalabs/common/t32/prog_spi.cmm
index bb48026..8993f5e 100644
--- a/board/habanalabs/common/t32/prog_spi.cmm
+++ b/board/habanalabs/common/t32/prog_spi.cmm
@@ -12,6 +12,9 @@
 &SPI_BASE=0x7ffcc43000
 &RAM_BASE=0x7ff0000000
 
+//SW Reset to have clean bus
+D.S axi:0x7FFCC4B474 %LE %Long 0x007FFE00
+
 /////////////////////////////////
 // SYStem up procedure
 /////////////////////////////////
